build/artifacts.props

```diff
@@ -113,6 +113,7 @@
     <PackageArtifact Include="Microsoft.AspNetCore.Server.Kestrel" Category="ship" Metapackage="true" />
     <PackageArtifact Include="Microsoft.AspNetCore.Session" Category="ship" Metapackage="true" />
     <PackageArtifact Include="Microsoft.AspNetCore.SpaServices" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.AspNetCore.SpaTemplates" Category="shipoob" />
```

Comment:
What happens differently for `shipoob` packages versus `ship` ones?

Reply:
Once the final packages are built packages marked as ship will be uploaded to nuget. shipoob packages are still produced by builds but are only available on the aspnetci drop share. Generally shipoob packages are released via a different mechanism.

Reply:
Will these ship on nuget.org when we release the runtime? If so, they should be `ship`. I wasn't sure so I picked "shipoob".

Reply:
Got it - thanks for the info.

The `SpaTemplates` one will ship on NuGet, since that's the only way customers can acquire it (it's not in the runtime package store and doesn't ship in the CLI). The others can be `shipoob`.

Reply:
To make sure this is completely clear:

* `Microsoft.AspNetCore.SpaTemplates` should go to NuGet.org (`ship`)
* But `Microsoft.DotNet.Web.Spa.ProjectTemplates` and the other packages can be `shipoob` as they are bundled into the CLI.